tasks/publisher.yml

```diff
@@ -14,7 +14,7 @@
 - name: Copy certificate key on publisher 
   ansible.builtin.copy:
     content: "{{ item.crt }}"
-    dest: /var/tmp/keys/mycert.crt
+    dest: /var/tmp/keys/"{{ item.crt }}"
```

Comment:
Is `item.crt` both the content of the file and the file name?

Reply:
My bad it's the name of repository i already fixed it.